### ISSUE TITLE
Automatically recheck template when related files are changed

### ISSUE THREAD

**Issue by wagenet** [lifeart/els-addon-typed-templates#76]
Right now if you change a component.ts file the associated .hbs file is not automatically rechecked.

**Comment by lifeart** [comment]
Yes, correct, likely we should add check on editor tab change, currently it’s working only on change

**Comment by wagenet** [comment]
Does that event fire when changing focus?
